src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -145,6 +146,10 @@ public override int Execute()
                     throw new GracefulException(string.Format(LocalizableStrings.WorkloadCacheDownloadFailed, e.Message), e, isUserError: false);
                 }
             }
+            else if (_skipManifestUpdate && usedRollback)
+            {
+                throw new GracefulException(LocalizableStrings.CannotCombineSkipManifestAndRollback, isUserError: true);
```

Comment:
Can we pass the two options names and use `string.format` when constructing the error? That way, if there's ever a rename (probably unlikely), we don't have to update the error message. And it also avoids issues if the translators make a mistake with a resource string.

Reply:
Just updated that, thanks for pointing it out!